Source/Assets/PlayFabEditorExtensions/Editor/Scripts/Panels/PlayFabEditorPackages.cs

```diff
@@ -46,13 +47,23 @@ public static void DrawPackagesMenu()
 #endif
         }
 
+        private static void DrawPubSubPrivatePreviewWarning()
+        {
+            GUILayout.BeginHorizontal();
+            GUILayout.Label(" PUBSUB IS IN PRIVATE PREVIEW.");
```

Comment:
You always inject spaces at the beginning of strings...

Reply:
Yes, I explicitly do it because Unity gives no margins to its GUILayout (based on the gui file), at this moment it's easier to add just a single space which helps determine the word wrap and split up the phrasing.